### PR TITLE
Abandoned Crates roll loot on open

### PR DESCRIPTION
**Pull request by cynic716**
## About The Pull Request

It just makes abandoned crates roll their loot upon being opened as opposed to being spawned, it's to fix some mobs being active while inside the crate, I suppose this means you could add active grenades and whatnot to the loottable if you wanted 

## Why It's Good For The Game
E
## Changelog
:cl:

fix: Abandoned crates will now roll their loot upon being opened. This should prevent certain crates exploding themselves roundstart.

/:cl:

### CODE REVIEW

File: end [hunk 1]
```diff
@@ -0,0 +1,171 @@
+[1mdiff --git a/code/modules/mining/abandoned_crates.dm b/code/modules/mining/abandoned_crates.dm[m
```
Comment (optimumtact) [inline end]: remove this

File: code/modules/mining/abandoned_crates.dm [hunk 2]
```diff
@@ -12,7 +12,7 @@
 	var/qdel_on_open = FALSE
 	tamperproof = 90
 
-/obj/structure/closet/crate/secure/loot/Initialize()
+/obj/structure/closet/crate/secure/loot/open(mob/living/user)
 	. = ..()
 	var/list/digits = list("1", "2", "3", "4", "5", "6", "7", "8", "9", "0")
```
Comment (optimumtact) [inline code/modules/mining/abandoned_crates.dm]: this code needs to be Initialize or the create will not be openable

Comment (optimumtact) [reply code/modules/mining/abandoned_crates.dm]: the code that should be in open starts with
`var/loot = rand(1,100) //100 different crates with varying chances of spawning`

**Comment by cynic716** [comment]
gotta redo this one

**Pull request closed without merging**